description/description.shacl.ttl

```diff
@@ -491,14 +491,17 @@
         a sh:PropertyShape ;
         sh:path ebucore:hasCastMember ;
         sh:nodeKind sh:Literal ;
-        sh:datatype xsd:string ;
+        sh:or ( 
+            [ sh:datatype xsd:string ]
+            [ sh:datatype ebucore:Cast ]
+        );
```

Comment:
@lucasotodegraeve We don't use ebucore:Cast in the data model, so I would keep this as xsd:string.